cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -124,7 +130,7 @@ def __init__(
 
     @cached_property
     def _executor(self) -> AsyncioExecutor:
-        return AsyncioExecutor()
+        return AsyncioExecutor.instance()
```

Comment:
I'd suggest adding a note here that we use a single instance because of threading problems in grpc and adding a link to the relevant grpc issue.

Reply:
+1 - I could have sworn I did that :-). \Added.